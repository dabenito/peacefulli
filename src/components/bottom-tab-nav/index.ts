export { Nav } from "./bottom-tab-nav";
