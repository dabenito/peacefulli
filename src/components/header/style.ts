import { StyleSheet } from 'react-native';
import { Color } from '../../constants/color-constants';

export const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        justifyContent: 'center',
    },
    text: {
        color: Color.purple,
        fontFamily: "lucida_grande_bold",
        fontSize: 36,
        fontWeight: "bold"
    }
});
