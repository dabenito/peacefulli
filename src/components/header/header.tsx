import React, { PureComponent } from "react";
import { View, Text } from "react-native";
import { styles } from "./style";

export interface HeaderProps {}

export interface HeaderState {}

export class Header extends PureComponent<HeaderProps, HeaderState> {
  constructor(props: HeaderProps) {
    super(props);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>TODO: Create Header Bar</Text>
      </View>
    );
  }
}
