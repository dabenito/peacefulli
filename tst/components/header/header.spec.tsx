import React from "react";
import { render, cleanup } from "@testing-library/react-native";
import { Header } from "../../../src/components/header";

describe("Header Tests", () => {
  afterEach(cleanup);

  // TODO: update this file when implementing header page
  it("loading header page", async () => {
    const header = render(<Header />);
    expect(header).toMatchSnapshot();
  });
});
