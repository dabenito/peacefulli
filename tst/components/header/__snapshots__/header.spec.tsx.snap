// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests loading header page 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    TODO: Create Header Bar
  </Text>
</View>
`;
